app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def new
+
+  end
+
+  def create
+    new_user = User.new(user_params)
+    if new_user.save
+      session[:user_id] = new_user.id
+      redirect_to "/dashboard"
+    else
+      flash[:failure] = "Something went wrong! Are you missing a required field? Are you already registered?"
+      # Eventually change this into validation error messages that rails already has
+      #Example: "ActiveRecord::RecordInvalid: Validation failed: Password confirmation doesn't match Password"
+    end
+  end
+
+  def show
+    @user = User.find(session[:user_id])
+  end
+
+  private
+  def user_params
+    params.permit(:email, :password, :password_confirmation)
```

Comment:
Do we need to permit `password_confirmation`?  I thought that adding a password confirmation was more for the user or Bcrypt gem, and less for us in our database validations.

Reply:
I believe this is to allow from the registration process
- A user goes to register
- They input email, password, and confirmation
- If their password does not match password confirmation, it sends back an error

Is this incorrect? I can try running it without, but I believe I have a sad path test for password and confirmation match.

Reply:
This WAS extraneous! Removed and pushed up changes